setup.py

```diff
@@ -37,7 +37,10 @@
         "pandas>=1.4,<1.5.4",
         "sqlalchemy>=1.4,<2.0.8",
```

Comment:
This dependency isn't really being tested, because pandas only supports SQLAlchemy 2.0 as of Pandas 2.0. So you might want to pin it to `sqlalchemy>=1.4,<2.0` for the time being.